README.md

```diff
@@ -1,7 +1,7 @@
 # Vet Clinic Database
 
```

Comment:
- [ ] It is a pair-parogramming project but i cannot see any commit from @Ibnballo1 in this pull request. @Ibnballo1 Kindly add a commit in this pull request and submit it again from the dashboard 👍